Please update the back-end tests, they fail as follows:

```
--- FAIL: TestCloudWatchMetrics (0.00s)
--
211 | --- FAIL: TestCloudWatchMetrics/When_calling_handleGetRegions (0.00s)
212 | metric_find_query_test.go:126:
213 | Error Trace:	metric_find_query_test.go:126
214 | Error:      	Not equal:
215 | expected: "ap-east-1"
216 | actual  : "af-south-1"
217 |  
218 | Diff:
219 | --- Expected
220 | +++ Actual
221 | @@ -1 +1 @@
222 | -ap-east-1
223 | +af-south-1
224 | Test:       	TestCloudWatchMetrics/When_calling_handleGetRegions
225 | metric_find_query_test.go:127:
226 | Error Trace:	metric_find_query_test.go:127
227 | Error:      	Not equal:
228 | expected: "ap-northeast-1"
229 | actual  : "ap-east-1"
230 |  
231 | Diff:
232 | --- Expected
233 | +++ Actual
234 | @@ -1 +1 @@
235 | -ap-northeast-1
236 | +ap-east-1
237 | Test:       	TestCloudWatchMetrics/When_calling_handleGetRegions
238 | metric_find_query_test.go:128:
239 | Error Trace:	metric_find_query_test.go:128
240 | Error:      	Not equal:
241 | expected: "ap-northeast-2"
242 | actual  : "ap-northeast-1"
243 |  
244 | Diff:
245 | --- Expected
246 | +++ Actual
247 | @@ -1 +1 @@
248 | -ap-northeast-2
249 | +ap-northeast-1
250 | Test:       	TestCloudWatchMetrics/When_calling_handleGetRegions
```

--- a/pkg/tsdb/cloudwatch/metric_find_query_test.go
+++ b/pkg/tsdb/cloudwatch/metric_find_query_test.go
@@ -123,9 +123,10 @@ func TestCloudWatchMetrics(t *testing.T) {
 		result, err := executor.handleGetRegions(context.Background(), simplejson.New(), &tsdb.TsdbQuery{})
 		require.NoError(t, err)
 
-		assert.Equal(t, "ap-east-1", result[0].Text)
-		assert.Equal(t, "ap-northeast-1", result[1].Text)
-		assert.Equal(t, "ap-northeast-2", result[2].Text)
+		assert.Equal(t, "af-south-1", result[0].Text)
+		assert.Equal(t, "ap-east-1", result[1].Text)
+		assert.Equal(t, "ap-northeast-1", result[2].Text)
+		assert.Equal(t, "ap-northeast-2", result[3].Text)
 	})
 
 	t.Run("When calling handleGetEc2InstanceAttribute", func(t *testing.T) {

